ament_clang_tidy/ament_clang_tidy/main.py

```diff
@@ -214,8 +214,14 @@ def start_subprocess(full_cmd):
     for filename in files:
         report[filename] = []
 
-    error_re = re.compile('(/.*?\\.(?:%s)):(\\d+):(\\d+): (?:warning:|error:)' %
-                          '|'.join(extensions))
```

Comment:
The inclusion of extensions in the compiled pattern appears to've been dropped with these changes. Is that intentional?

Reply:
It is intentional. The pattern before was matching a filename and then an extension before moving to the `:<row>:<col>` sections of the output. The new regex will match paths that start with `/home/`, then any number of characters that is not a `:` which indicates the start of the `:<row>:<col>` block. I believe this matches the same strings as before, and is slightly less complex.

Reply:
> The new regex will match paths that start with /home/, then any number of characters that is not a `:`...

You know, I saw this in the regexp and didn't question it, and it is already in the code but.. does this mean that errors outside of the /home/ path will not be captured?

Some of our builds occur in `/root/` or `/src/`.

Reply:
It does mean those won't be captured. What are the cases in which builds happen under `/root` or `/src`? Happy to swap this regex to be more accommodating. Should we match `(?:/home|/root|/src)`, or should it be more permissive and require a preceding `/` but not make any other assumptions about the path?

Reply:
Resolved in 5d5e2eda